app/src/services/object.js

```diff
@@ -99,6 +100,7 @@ const service = {
       .modify('filterUserId', params.userId)
       .modify('filterOriginalName', params.originalName)
       .modify('filterMimeType', params.mimeType);
+    // .modify('filterMetadata', params.metadata);
```

Comment:
If we're not implementing the search side of things just yet, let's not add this in as a comment yet.